### PR TITLE
add new commands

### PR DESCRIPTION
**Pull request by DarkPark**

### CODE REVIEW

File: src/mtx/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -9,6 +319,244 @@ class CommandBinaryBuffer extends BinaryBuffer {
...
+            dateTime.seconds
+        );
+    }
+
+    // does not work fully for now
+    static getDateTimeFromDate ( date: Date ): IDateTime {
+        const dateTime: IDateTime = {
+            year: date.getFullYear() - 2000,
+            month: date.getMonth() + 1,
+            day: 0, // todo
```
Comment (DeadByElpy) [inline src/mtx/CommandBinaryBuffer.ts:339]: date.getDay?

File: src/mtx/CommandBinaryBuffer.ts [hunk 2]
```diff
@@ -1,4 +1,314 @@
...
+    VBAT: 0x00200000,
+    POWER_THRESHOLD_T0: 0x00400000,
+    POWER_THRESHOLD_T1: 0x00800000,
+    POWER_THRESHOLD_T2: 0x01000000,
+    POWER_THRESHOLD_T3: 0x02000000,
+    SALDO_D: 0x20000000,
+    SCROLL_D: 0x80000000
+};
+
+const displaySetExtMask = {
```
Comment (DeadByElpy) [inline src/mtx/CommandBinaryBuffer.ts:228]: `displaySetExtMask` and `displaySetMask` have common part (displaySetMask without SCROLL_D) maybe create general/base mask `displaySetMaskBase`?

File: src/mtx/CommandBinaryBuffer.ts (hunk 1 above)
Comment (DarkPark) [reply src/mtx/CommandBinaryBuffer.ts]: done

File: src/mtx/CommandBinaryBuffer.ts (hunk 2 above)
Comment (DarkPark) [reply src/mtx/CommandBinaryBuffer.ts:204]: done

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**